[slack] Add help message for Slack users

Harshg999 opened the pull request:
## What changes were proposed in this pull request?

- Help message for better usability

<img width="701" alt="Screenshot 2021-05-30 at 10 30 49 PM" src="https://user-images.githubusercontent.com/42064744/120113141-b6bc9700-c196-11eb-802c-98f1eae4f92f.png">

romainr commented on desktop/core/src/desktop/lib/botserver/views.py:
```diff
@@ -95,13 +95,62 @@ def parse_events(request, event):
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:page_facing_up: Share directly from Editor:* Create a query gist, select channel and share your query directly from the Hue Editor window. Requires `share_from_editor` flag enabled."
```
We should just turn it on by default

Harshg999 replied:
Will send a PR for it!

romainr commented on desktop/core/src/desktop/lib/botserver/views.py:
```diff
@@ -95,13 +95,62 @@ def parse_events(request, event):
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": "*:question: Surfacing Query bank:* Type `@Hue Bot queries` to explore a list of important queries from the latest query bank."
```
No need to have 'Bot'?

Harshg999 replied:
FIxed and will fix in the latest app manifest too!

romainr reviewed (approved):
Maybe no icons to keep it less noisy?

Also a link to the docs?

The name of each section is more or less the same as its description

Harshg999 commented:
- Updated unit tests
- Updated preview below
- `much more` --> docs link
<img width="684" alt="Screenshot 2021-05-31 at 1 01 31 AM" src="https://user-images.githubusercontent.com/42064744/120117633-be3a6b00-c1ab-11eb-8207-13ca2bc1370b.png">

romainr reviewed (approved):
Sweet! 

Let's ship it and iterate!
